common/utils/src/main/resources/error/error-classes.json

```diff
@@ -3748,6 +3748,12 @@
     },
     "sqlState" : "0A000"
   },
+  "UNSUPPORTED_ENCODER" : {
+    "message" : [
+      "Found unsupported encoder. Try switching to expression encoder."
+    ],
+    "sqlState" : "0A000"
```

Comment:
The category `0A` belongs to:
```
    "0A": "feature not supported",
```
see `error-categories.json`. Any reasons to choose this one? How about `46` or `42`?

Reply:
I have changed the category to `42` as I believe the error is possible to happen only if a user defines implicit `Encoder` for which an implicit `Encoder` does not already exist. I experimented with trying to use types `Int` or `String`, but realised that we already have implicit values defined and in those cases `.toDS()` call was marked as unresolved and the code did not even compile. `KryoData` seems to not have an implicit Encoder defined, so the added test is possible to happen. So I am not sure if I should change error name and/or message accordingly?